src/features/products/screen/productItem.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Image, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import Icon from 'react-native-vector-icons/Entypo';
+
+import { Product } from 'network/models/product-models';
+
+const ProductItem: React.FunctionComponent<{
+  item: Product;
+  isLast: boolean;
+  onItemPress: () => void;
+  onLikePress: () => void;
+  onBuyPress: () => void;
+}> = ({ item, isLast, onItemPress, onLikePress, onBuyPress }) => {
+  return (
+    <TouchableOpacity
+      style={isLast ? styles.containerLast : styles.container}
+      onPress={() => onItemPress()}>
+      <Image source={{ uri: item.pictures ? item.pictures[0] : '' }} style={styles.image} />
+      <View style={styles.data}>
+        <Text style={styles.name}>{item.title}</Text>
+        <Text numberOfLines={1} style={styles.status}>
+          {item.state}
+        </Text>
+        <Text style={styles.price}>{item.unitPrice}</Text>
+        <TouchableOpacity style={styles.likeButton} onPress={() => onLikePress()}>
+          <Icon name="heart-outlined" size={24} color="#000000" />
+        </TouchableOpacity>
+        <TouchableOpacity style={styles.buyButton} onPress={() => onBuyPress()}>
+          <Text style={styles.buyText}>Add to cart</Text>
+        </TouchableOpacity>
+      </View>
+    </TouchableOpacity>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    height: 140,
+    width: '100%',
+    padding: 10,
+    flexDirection: 'row',
+    borderBottomWidth: 1,
+    borderBlockColor: '#000000',
+  },
+  containerLast: {
+    height: 150,
+    width: '100%',
+    padding: 10,
+    flexDirection: 'row',
+  },
```

Comment:
You may want to apply only the different styles to the last container. Remember the `style` prop accepts an array of styles, so you could do: `[styles.container, isLast ? styles.containerLast : ...[]]`

Reply:
Not blocker tho